modules/unmanaged-operators.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture-installation.adoc
+// * updating/updating-cluster-between-minor.adoc
+
+[id="unmanaged-operators_{context}"]
+= Unmanaged Operators
+
+The _management state_ of an Operator determines whether an Operator is actively
+managing the resources for its related component in the cluster as designed. If
+an Operator is set to an unmanaged state, it does not respond to changes in
+configuration nor does it receive updates.
+
+While this can be helpful in non-production clusters or during debugging,
+Operators in an unmanaged state are unsupported and the cluster administrator
+assumes full control of the individual component configurations and upgrades.
+Depending on the method the Operator was set to unmanaged, it can also render
+the entire cluster unsupported.
+
+An Operator's management state is accessible using the following methods:
+
+* **Cluster Version Operator (CVO) overrides**
++
+The CVO configuration includes access to the `spec.overrides` parameter, which
+allows administrators to provide a list of overrides to the CVO's behavior for a
+component. For example:
++
+[source,yaml]
+----
+spec:
+  channel: stable-4.4
+  clusterID: 83062003-0ede-460e-9dea-3330cc8a37b3
+  overrides:
+  - group: apps/v1
+    kind: Deployment
+    name: network-operator
+    namespace: openshift-network-operator
+    unmanaged: true
+----
++
+The `spec.overrides[].unmanaged` parameter is set to `false` by default, however
+setting it to `true` blocks cluster upgrades and alerts the administrator that a
+CVO override has been set:
++
+----
+Disabling ownership via cluster version overrides prevents upgrades. Please remove overrides before continuing.
+----
++
+[WARNING]
+====
+Setting CVO overrides puts the entire cluster in an unsupported state. Reported
+issues must be reproduced after removing the overrides for support to proceed.
+====
+
+* **Individual Operator configuration**
++
+Individual Operators have a `managementState` parameter in their configuration,
+which does not apply to the CVO. This can be accessed in different ways,
+depending on the Operator. For example, the Cluster Logging Operator
+accomplishes this by modifying a Custom Resource (CR) that it manages, while the
+Samples Operator uses a cluster-wide configuration resource.
++
+The following management states are available with this method:
```

Comment:
What are these based on (what are we documenting)? 
Are we documenting an example using the CR (like the Cluster Logging Operator), or the Samples Operator?

Reply:
https://github.com/openshift/openshift-docs/pull/21553#issuecomment-619930445

CLO and Samples Operator were just mentioned as examples, since I noticed the procedure for how they're set to unmanaged differed slightly.